src/cli/proot.h

```diff
@@ -269,7 +269,7 @@ Copyright (C) 2021 PRoot Developers, licensed under GPL v2 or later.",
 	  .arguments = {
 		{ .name = "-P", .separator = ' ', .value = "string" },
 		{ .name = NULL, .separator = '\0', .value = NULL } },
-	  .handler = handle_option_p,
+	  .handler = handle_option_P,
```

Comment:
Yeah, looks like it was confused...instead of the Python extension, that is for the port extension.